streams/src/test/java/org/apache/kafka/streams/kstream/internals/CogroupedKStreamImplTest.java

```diff
@@ -0,0 +1,127 @@
+package org.apache.kafka.streams.kstream.internals;
+
+import org.apache.kafka.common.errors.InvalidTopicException;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.kstream.CogroupedKStream;
+import org.apache.kafka.streams.kstream.KStream;
+import org.apache.kafka.streams.kstream.KStreamBuilder;
+import org.apache.kafka.streams.kstream.Merger;
+import org.apache.kafka.streams.kstream.SessionWindows;
+import org.apache.kafka.streams.kstream.TimeWindows;
+import org.apache.kafka.test.KStreamTestDriver;
+import org.apache.kafka.test.MockAggregator;
+import org.apache.kafka.test.MockInitializer;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CogroupedKStreamImplTest {
+
+    private static final String TOPIC_1 = "topic-1";
+    private static final String TOPIC_2 = "topic-2";
+    private static final String INVALID_STORE_NAME = "~foo bar~";
+    private final KStreamBuilder builder = new KStreamBuilder();
+    private CogroupedKStream<String, String> cogroupedStream;
+    private KStreamTestDriver driver = null;
+    
+    private final Merger<String, String> sessionMerger = new Merger<String, String>() {
+        @Override
+        public String apply(final String aggKey, final String aggOne, final String aggTwo) {
+            return null;
+        }
+    };
+
+    @Before
+    public void before() {
+        final KStream<String, String> stream = builder.stream(Serdes.String(), Serdes.String(), TOPIC_1);
+        cogroupedStream = stream.groupByKey(Serdes.String(), Serdes.String()).cogroup(MockAggregator.TOSTRING_ADDER);
+    }
+
+    @After
+    public void cleanup() {
+        if (driver != null) {
+            driver.close();
+        }
+        driver = null;
```

Comment:
nit: no need to set `driver` to `null` each test gets it's own instance of the class, so it will always be null for the next test.